### ISSUE TITLE
use of depreciated methods in options.js

### ISSUE THREAD

**Issue by nithssh** [ganigeorgiev/hide-ratings-extension#4]
in ln34 of options.js:
```javascript
        chrome.storage.sync.set(data, function () {
            chrome.tabs.getSelected(function (tab) {
                if (tab && tab.url) {
                    chrome.tabs.sendMessage(tab.id, data);
                }
            });
```
should be changed to:
```javascript
        chrome.storage.sync.set(data, function () {
            chrome.tabs.query({active: true}, function (tab) {
                if (tab && tab.url) {
                    chrome.tabs.sendMessage(tab.id, data);
                }
            });
```
as the tabs.getSelected method is depreciated on chrome (and completely unimplemented in Firefox).

**Comment by nithssh** [comment]
![image](https://user-images.githubusercontent.com/22272753/82249525-8f1dd680-9967-11ea-9050-f318197058a7.png)
https://developer.chrome.com/extensions/tabs

**Comment by ganigeorgiev** [comment]
@Dem1se Thanks for reporting the issue.

Actually `sendMessage` should be called for all tabs, not only the active one. In addition, the `tab.url` check is not necessary and by removing it there is also no need for the `activeTab` permission.

The above issues have been fixed in [v1.6.2 release](https://github.com/ganigeorgiev/hide-ratings-extension/releases/tag/v1.6.2).
